docs/sources/explore/query-management.md

```diff
@@ -73,29 +73,3 @@ You can customize the query history in the Settings tab. Options are described i
 | Change the default active tab | Query history tab |
 
 > **Note:** Query history settings are global, and applied to both panels in split mode.
-
-## Prometheus-specific Features
```

Comment:
We don't have Prometheus-specific features anymore. These features are now available also in Dashboards and documented in Prometheus docs: https://grafana.com/docs/grafana/latest/datasources/prometheus/query-editor/#metrics-browser